pkg/printers/printers.go

```diff
@@ -46,57 +46,41 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 			Decoders: decoders,
 		}
 
-	case "template", "go-template":
+	case "templatefile", "go-template-file", "jsonpath-file":
```

Comment:
Eventually the logic from this fallthrough block is going to need to be handled in the actual `*Flags` structs themselves.